app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
```

Comment:
When importing multiple classes or modules, it's recommended to use parentheses for better readability according to the checklist.